src/libLookup/Lookup.cpp

```diff
@@ -1878,6 +1838,76 @@ bool Lookup::ProcessSetOfflineLookups(const std::vector<unsigned char>& message,
     return true;
 }
 
+bool Lookup::ProcessRaisePowSubmission(
+    const std::vector<unsigned char>& message, unsigned int offset,
+    const Peer& from)
+{
+    LOG_MARKER();
+    bPOWSubmission = true;
```

Comment:
what is the meaning of b in bPOWSubmission?

Reply:
It's just naming, "b" stands for "boolean"